### PR TITLE
add option to update results of query-tests

### PR DESCRIPTION
**Pull request by candux**
I didn't found any way to update the results of query-tests so far. Add UPDATE=true option like for render tests.
I reformatted all results with JSON.stringify()
## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Suggest a changelog category: bug/feature/docs/etc. or "skip changelog".: skip changelog

### CODE REVIEW

File: test/integration/query/query.test.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {deepEqual} from '../lib/json-diff';
 import st from 'st';
 import http from 'http';
 import path from 'path';
+const fs = require('fs');
```
Comment (HarelM) [inline test/integration/query/query.test.ts:11]: Please don't use require, use import instead.

File: test/integration/query/tests/fill/fill-pattern/expected.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
           ],
           [
             -75.13824462890625,
-            39.93501296038252
+            39.93501296038255
```
Comment (HarelM) [inline test/integration/query/tests/fill/fill-pattern/expected.json:53]: These differences sound not be part of this PR

File: test/integration/query/query.test.ts (hunk 1 above)
Comment (candux) [reply test/integration/query/query.test.ts]: oops. I changed id

File: test/integration/query/tests/fill/fill-pattern/expected.json (hunk 2 above)
Comment (candux) [reply test/integration/query/tests/fill/fill-pattern/expected.json]: I changed them back. Apparently they are equal in FP-Arithmetic

File: test/integration/query/query.test.ts [hunk 3]
```diff
@@ -173,6 +174,11 @@ describe('query tests', () => {
 
             const actual = await page.evaluate(performQueryOnFixture, fixture);
 
+            // update expected.json if UPDATE=true is passed and the test fails
+            if (process.env.UPDATE && !deepEqual(actual, fixture.expected)) {
```
Comment (HarelM) [inline test/integration/query/query.test.ts:178]: I would consider storing the results of deepEqual in a variable to avoid running it twice...?

Comment (candux) [reply test/integration/query/query.test.ts]: since it only runs twice when UPDATE=true is set, it doesn't make a big difference. I changed it anyway for better style

**Comment by candux** [comment]
> Please run lint before pushing the code to avoid CI failures...
sorry, Linting problems are now fixed

**Review by HarelM — APPROVED** [review]
